pre-release.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ $# < 4 ]]
+then
+  echo "Usage $(basename $0) splunk_old_version splunk_new_version otel_instrumentation_old_version otel_instrumentation_new_version [otel_old_version] [otel_new_version]"
```

Comment:
I feel like this is a lot of similarly named arguments to read horizontally.  Maybe split to new lines for readability?  Alternatively, if we think this is always manual we could prompt for them...

Reply:
Or we could use getopts.

Reply:
I thought about using getops or case/shift, but then I decided to keep it simple...
But you're right, it would be more readable that way -- will refactor.

> Alternatively, if we think this is always manual we could prompt for them...

I'm hoping that sometime in the future this script (or its future version) will be executed by an automatic process, so I wanted to avoid prompts. Although I don't know how distant that future is 😅